src/cloud_client/dropbox/parameters/upload.rs

```diff
@@ -0,0 +1,15 @@
+use derive_builder::Builder;
+use serde::{Deserialize, Serialize};
+use std::path::PathBuf;
+
+#[derive(Serialize, Deserialize, Builder)]
+pub struct UploadParameters {
+    path: PathBuf,
+    #[builder(default)]
+    #[serde(skip_serializing_if = "Option::is_none")]
+    // "add", "overwrite", "update"
+    mode: Option<String>,
```

Comment:
I think this `String` can be replaced with `enum`

Reply:
Done